front/pages/api/v1/w/[wId]/data_sources/[name]/tables/index.ts

```diff
@@ -98,7 +99,8 @@ async function handler(
           status_code: 500,
           api_error: {
             type: "internal_server_error",
-            message: "Failed to get tables.",
+            message: "Failed to retrieve tables.",
+            data_source_error: tablesRes.error,
```

Comment:
is `data_source_error` the right name here ?

Reply:
We're still in the context of data sources so I think that's fine that's just a way to surface more granular errors.

We support `data_source_error` `run_error` `app_error` and `connectors_error` as such additional error types

Reply:
Fair 👍